website/source/docs/builders/amazon-chroot.html.md

```diff
@@ -128,12 +128,6 @@ each category, the available configuration keys are alphabetized.
     source AMI will be attached. This defaults to "" (empty string), which
     forces Packer to find an open device automatically.
 
--   `enhanced_networking` (boolean) - Enable enhanced
```

Comment:
do these no longer apply to the chroot builder? I think this might just be an oversight

Reply:
I looked at the chroot builder on master and it isn't actually using the "ehnahced_networking" variable anywhere except for StepSourceAMIInfo; which doesn't actually do any of the networking changes.  Maybe this was an oversight, but I removed the docs rather than changing the code functionality.

Reply:
On second look you're right -- it was just being passed inconsistently.